TalismanSociety/talisman-web
# feat: vDOT staking

# Description

- Added vDOT staking through Bifrost SLPx protocol.
- Fixed Styling inconsistencies between Stake and Positions rows.
- Added chain logo to asset logo.

## Type of change


### 🧪 **How to test:**

- Get some DOT on Bifrost
- Go to `/staking/positions`
- Select DOT asset on Bifrost SLPx
- Stake it!
- ???
- Profit.
- Check you wallet, it should have the correct DOT to vDOT rate of tokens.

### 🖼️ **Screenshots:**


https://github.com/user-attachments/assets/298276a1-3aa2-4ef1-b8de-26c281edb234


https://github.com/user-attachments/assets/3cc6904b-de24-44e0-8fde-cd317ed2bc1c
